subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -319,22 +323,38 @@ int lll_deinit(void)
 
 int lll_csrand_get(void *buf, size_t len)
 {
+#if defined(CONFIG_ENTROPY_NRF5_RNG)
 	return entropy_get_entropy(dev_entropy, buf, len);
+#else
+	return 0;
+#endif
```

Comment:
For this and the other functions in this commit, shouldn't we return `-ENOTSUP` instead?

As per the https://docs.zephyrproject.org/latest/develop/api/design_guidelines.html if a function is not implemented or supported, it should return that error. 

Returning `0` when trying to get a value seems like a bad idea.

Reply:
Added comment.